internal/endtoend/testdata/pg_timezone_names/sqlc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "2",
+  "sql": [
+    {
+      "engine": "postgresql",
+      "schema": "query.sql",
+      "queries": "query.sql",
+      "gen": {
+        "go": {
+          "sql_package": "pgx/v4",
+          "package": "querytest",
+          "out": "go_pgx"
+        }
+      }
+    },
```

Comment:
@kyleconroy I found that undoing the nesting (and using multiple `sql` array elements, made these things a lot easier to test.